### ISSUE TITLE
Unknown compute type error 

### ISSUE THREAD

**Issue by swinner95** [Microsoft/vscode-tools-for-ai#140]
When I select another workspace or when I try viewing computes in another workspace/subscription,  I get the error below:

"Internal extension error: Unknown compute type encountered: DataLakeAnalytics"

**Comment by LittleLittleCloud** [comment]
By selecting another workspace/subscription, do you mean whatever selections can trigger the error or just some specific ones.

**Comment by swinner95** [comment]
So I realized the issue is that the AML for VS Code extension currently supports only the attachment of AKS, Virtual Machine and HDInsight. It does not currently support the attachment of computes for 'Azure Databricks' and 'Azure Lake Analytics'. Should I submit a different issue for a feature request?

**Comment by LittleLittleCloud** [comment]
Sounds like a feature that will have to implement in the future. But based on the information I have I'm not so sure about that. Could you discuss it with @greazer .

**Comment by shsuman** [comment]
@swinner95 you are right, currently only HDI, AKS, AML and VM are supported for attach. However to support the attach of all those other computes, the sdk needs to support it. I am sure DataBricks is supported but I am not sure about Azure Lake Analytics.  I can see that a compute type of Data Lake Analytics is supported. Are they the same ?
